account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/Stampler.java

```diff
@@ -0,0 +1,211 @@
+package com.alchemy.aa;
+
+import com.alchemy.aa.core.TEKManager;
+import com.alchemy.aa.core.Utilities;
+import com.alchemy.aa.core.exceptions.NoInjectedBundleException;
+import com.alchemy.aa.core.exceptions.StamperNotInitializedException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.google.common.primitives.Bytes;
+import com.google.crypto.tink.subtle.Base64;
+import com.google.crypto.tink.subtle.EllipticCurves;
+
+import java.nio.ByteBuffer;
+import java.security.GeneralSecurityException;
+import java.security.KeyFactory;
+import java.security.interfaces.ECPrivateKey;
+import java.security.interfaces.ECPublicKey;
+import java.security.Signature;
+import java.util.Locale;
+import org.bitcoinj.base.Base58;
+import org.bouncycastle.jce.ECNamedCurveTable;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.bouncycastle.jce.spec.ECPublicKeySpec;
+
+public class Stampler {
+
+  public record Stamp(
+      String stampHeaderName,
+      String stampHeaderValue
+  ){}
+
+  public record APIStamp(
+      String publicKey,
+      String scheme,
+      String signature
+  ){}
+
+  public static String toJson( APIStamp apiStamp ) throws JsonProcessingException {
+    ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
+    return ow.writeValueAsString(apiStamp);
+  }
+
+  private TEKManager tekManager;
+
+  private char[] bundlePrivateKey;
+  private char[] bundlePublicKey;
+  public Stampler(TEKManager tekManager) {
```

Comment:
see comment below (above?) about creation of a stamper with a TEK that's externally managed

Reply:
I think having both constructors probably makes sense